tox.ini

```
@@ -103,9 +103,22 @@ passenv =
   PRE_COMMIT_HOME
 
 [testenv:docs]
-whitelist_externals = make
-deps = sphinx
-commands = make -C docs/docsite htmldocs
+description = Invoke sphinx-build to build the HTML docs
+basepython = python3
+deps = -r{toxinidir}/docs/requirements.txt
+passenv = *
+usedevelop = False
+commands =
+    python -m sphinx \
+      -a -n -W \
+      -b html --color \
+      -d "{toxworkdir}/.docstree" \
```

this should probably be
```suggestion
      -d "{temp_dir}/.docstree" \
```